misc/helm-charts/operator/README.md

```diff
@@ -107,165 +107,48 @@ The following table lists the configurable parameters of the Materialize operato
-| `operator.clusters.sizes.50cc.memory_limit` |  | ``"7762MiB"`` |
-| `operator.clusters.sizes.50cc.scale` |  | ``1`` |
-| `operator.clusters.sizes.50cc.workers` |  | ``1`` |
-| `operator.clusters.sizes.512C.cpu_exclusive` |  | ``true`` |
-| `operator.clusters.sizes.512C.cpu_limit` |  | ``62`` |
-| `operator.clusters.sizes.512C.credits_per_hour` |  | ``"512"`` |
-| `operator.clusters.sizes.512C.disk_limit` |  | ``"962560MiB"`` |
-| `operator.clusters.sizes.512C.memory_limit` |  | ``"481280MiB"`` |
-| `operator.clusters.sizes.512C.scale` |  | ``16`` |
-| `operator.clusters.sizes.512C.workers` |  | ``62`` |
-| `operator.clusters.sizes.600cc.cpu_exclusive` |  | ``true`` |
-| `operator.clusters.sizes.600cc.cpu_limit` |  | ``12`` |
-| `operator.clusters.sizes.600cc.credits_per_hour` |  | ``"6"`` |
-| `operator.clusters.sizes.600cc.disk_limit` |  | ``"186301MiB"`` |
-| `operator.clusters.sizes.600cc.memory_limit` |  | ``"93150MiB"`` |
-| `operator.clusters.sizes.600cc.scale` |  | ``1`` |
-| `operator.clusters.sizes.600cc.workers` |  | ``12`` |
-| `operator.clusters.sizes.6400cc.cpu_exclusive` |  | ``true`` |
-| `operator.clusters.sizes.6400cc.cpu_limit` |  | ``62`` |
-| `operator.clusters.sizes.6400cc.credits_per_hour` |  | ``"64"`` |
-| `operator.clusters.sizes.6400cc.disk_limit` |  | ``"962560MiB"`` |
-| `operator.clusters.sizes.6400cc.memory_limit` |  | ``"481280MiB"`` |
-| `operator.clusters.sizes.6400cc.scale` |  | ``2`` |
-| `operator.clusters.sizes.6400cc.workers` |  | ``62`` |
-| `operator.clusters.sizes.800cc.cpu_exclusive` |  | ``true`` |
-| `operator.clusters.sizes.800cc.cpu_limit` |  | ``16`` |
-| `operator.clusters.sizes.800cc.credits_per_hour` |  | ``"8"`` |
-| `operator.clusters.sizes.800cc.disk_limit` |  | ``"248402MiB"`` |
-| `operator.clusters.sizes.800cc.memory_limit` |  | ``"124201MiB"`` |
-| `operator.clusters.sizes.800cc.scale` |  | ``1`` |
-| `operator.clusters.sizes.800cc.workers` |  | ``16`` |
-| `operator.clusters.sizes.mz_probe.cpu_exclusive` |  | ``false`` |
-| `operator.clusters.sizes.mz_probe.cpu_limit` |  | ``0.1`` |
-| `operator.clusters.sizes.mz_probe.credits_per_hour` |  | ``"0.00"`` |
-| `operator.clusters.sizes.mz_probe.disk_limit` |  | ``"1552MiB"`` |
-| `operator.clusters.sizes.mz_probe.memory_limit` |  | ``"776MiB"`` |
-| `operator.clusters.sizes.mz_probe.scale` |  | ``1`` |
-| `operator.clusters.sizes.mz_probe.workers` |  | ``1`` |
-| `operator.features.authentication` |  | ``false`` |
-| `operator.features.consoleImageTagMapOverride` |  | ``{}`` |
-| `operator.features.createBalancers` |  | ``true`` |
-| `operator.features.createConsole` |  | ``true`` |
-| `operator.image.pullPolicy` |  | ``"IfNotPresent"`` |
-| `operator.image.repository` |  | ``"materialize/orchestratord"`` |
-| `operator.image.tag` |  | ``"v0.127.0-dev.0"`` |
+| `operator.features.authentication` | Whether to enable environmentd rbac checks TODO: this is not yet supported in the helm chart | ``false`` |
```

Comment:
Just curious -- will this be supported by the tentative January release or is it a later feature?